templates/partial/search_result.html

```diff
@@ -27,28 +27,22 @@ <h3 class="govuk-heading-m govuk-!-margin-bottom-2">
             {{ result.description|truncate_snippet:300|markdown:3 }}
           </div>
         {% endif %}
-        <dl class="govuk-summary-list govuk-summary-list--no-border">
-          <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key">Refresh period:</dt>
-            <dd class="govuk-summary-list__value">
-              TBC
-            </dd>
-          </div>
-          <div class="govuk-summary-list__row">
-            <dt class="govuk-summary-list__key">Domain name</dt>
-            <dd class="govuk-summary-list__value">
-              {{result.metadata.domain_name}}
-            </dd>
-          </div>
+        <ul class="govuk-list govuk-body" id="metadata-property-list">
+          <li>
+            <span class="govuk-!-font-weight-bold">Refresh period:</span>
+            <span>TBC</span>
```

Comment:
We probably need a ticket to start populating this 🤔

Reply:
https://github.com/ministryofjustice/find-moj-data/issues/516